conda_build/develop.py

```diff
@@ -136,12 +142,11 @@ def execute(
     uninstall: bool = False,
 ) -> None:
     if not isdir(prefix):
-        sys.exit(
-            f"""\
-Error: environment does not exist: {prefix}
-#
-# Use 'conda create' to create the environment first.
-#"""
+        raise CondaBuildUserError(
+            f"""Error: environment does not exist: {prefix}
+            \n
+            Use 'conda create' to create the environment first.
+            """
```

Comment:
The indentation will be preserved in this error message so either use `textwrap.dedent` or my preferred style that avoids an import and yet offers similar readability:

```suggestion
            f"Error: environment does not exist: {prefix}\n"
            f"\n"
            f"Use 'conda create' to create the environment first."
```